code/src/pages/cart/CartPage.scss

```diff
@@ -0,0 +1,78 @@
+@import "@design-system";
+
+.spa-cart-page {
+  padding: spacing(10);
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    gap: spacing(5);
+  }
+
+  &__items,
+  &__order-summary {
+    padding: spacing(5);
+    border: 1px solid var(--spa-order-summary-border-color, $white);
+    border-radius: spacing(1);
+    background-color: var(--spa-order-summary-background-color, $white);
+  }
+
+  &__items {
+    width: 60%;
+
+    h1 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-cart-title-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__order-summary {
+    width: 35%;
+
+    h2 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-summary-border-color, $gray-light);
+      padding-bottom: spacing(3);

```

Comment:
Use existing typography variants for styling. You don't need directly make styles for it. Also, give className to it. It's bad practice to use html attributes as a selectors

Reply:
> Use existing typography variants for styling. You don't need directly make styles for it. Also, give className to it. It's bad practice to use html attributes as a selectors

which elements I should to remove here ?

Reply:
I suggest to define fs-2 or 3xl in this object and use it like variant. As you can see, in your code this font size repeats at least 2 times
![image](https://github.com/idx-academy/spa-orders/assets/118522791/2c819d94-bdea-49de-95f4-0b4fb5e61919)

Reply:
> I suggest to define fs-2 or 3xl in this object and use it like variant. As you can see, in your code this font size repeats at least 2 times ![image](https://private-user-images.githubusercontent.com/118522791/347163673-2c819d94-bdea-49de-95f4-0b4fb5e61919.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1NjQ3OTIsIm5iZiI6MTcyMDU2NDQ5MiwicGF0aCI6Ii8xMTg1MjI3OTEvMzQ3MTYzNjczLTJjODE5ZDk0LWJkZWEtNDlkZS05NWY0LTBiNGZiNWU2MTkxOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOVQyMjM0NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZmVlNDQyMmYwMmRiM2RkMWJlN2FjZjQ4YWE1MjAxMGY4NjE5NDJiN2FhMzdiNzllYmVhNGY2ZGQ1NjU0ZmM4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.NcVrD7m638X6psxhmTRh-KCdnI2ve_k-BjBXKedu8fQ)

changed but for summary I left `spacing` because it should be in one line. this value unsuitable for that
<img width="184" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/fbc6ae70-6b74-4a26-8b96-f2ec63ba733c">